docs/operator_guides/0_running_an_operator.md

```diff
@@ -152,6 +152,65 @@ If you don't have Holesky ETH, these are some useful faucets:
 ./operator/build/aligned-operator start --config ./config-files/config-operator.yaml
 ```
 
+### Run Operator using Systemd
+
+To manage the Operator process on Linux systems, we recommend use Systemd with the following configuration:
+
+```shell
+# aligned-operator.service
+
+[Unit]
+Description=Aligned Operator
+After=network.target
+
+[Service]
+Type=simple
+User=<user>
+WorkingDirectory=<path_to_aligned_layer_repository>
+ExecStart=<path_to_aligned_layer_repository>/operator/build/aligned-operator start --config <path_to_operator_config>
+Restart=always
+RestartSec=1
+StartLimitBurst=100
+
+[Install]
+WantedBy=multi-user.target
+```
+
+{% hint style="info" %}
+`aligned-operator.service` is just an arbitrary name. You can name your service as you wish, following the format `<service-name>.service`.
+{% endhint %}
+
+Once you have configured the `aligned-operator.service` file, you need to run the following commands:
+
+```shell
+sudo ln -s <path_to_service_file>/aligned-operator.service /etc/systemd/system/aligned-operator.service
+sudo systemctl enable --now aligned-operator.service
+```
+
+{% hint style="warning" %}
+All paths must be absolute.
+{% endhint %}
```

Comment:
Also not rendering.

Reply:
We use gitbook to render the docs, and that is the reason why we use that format. Rendering works different in gitbook and github